scipy/integrate/quadpack.py

```diff
@@ -330,9 +330,15 @@ def quad(func, a, b, args=(), full_output=0, epsabs=1.49e-8, epsrel=1.49e-8,
     (0.0, 0.0) 
 
     """
+    if b < a:
```

Comment:
How about:
```
a, b = min(a, b), max(a, b)
```
instead of adding this clause?

Reply:
Not quite, because below it integrates either f(x) or -f(x), depending on whether a<b or not.